va/va_enc_av1.h

```diff
@@ -575,7 +578,12 @@ typedef struct  _VAEncPictureParameterBufferAV1
      */
     uint8_t     ref_frame_idx[7];
 
-    uint8_t     reserved8bits0;
+    /** \brief When hierachical_level_plus1 > 0, hierachical_level_plus1-1 indicates
+     *the current frame's level.when it > 0. B1, B2 setting in CodingType can be treated as B,
+     *hirachical level is determined by this variable.When hierachical_level_plus1  == 0,
+     *hierarchical level information still comes from coding_type.
+     */
+    uint8_t     hierarchical_level_plus1;
```

Comment:
hi @RachelChengRJ could it be replaced by VAEncMiscParameterTemporalLayerStructure?

Reply:
Hi Carl, as we offline discussed, temporal layer info only provide  mini GOP structure inside whole sequence, not each frame’s layer info directly, so basically this change is fine, can be merged.